src/overlays/actors/ovl_En_Fall/z_en_fall.c

```diff
@@ -759,67 +807,74 @@ void EnFall_LodMoon_DrawWithLerp(Actor* thisx, GlobalContext* globalCtx) {
         Matrix_InsertXRotation_s(thisx->shape.rot.x, MTXMODE_APPLY);
         Matrix_InsertZRotation_s(thisx->shape.rot.z, MTXMODE_APPLY);
     }
+
     EnFall_LodMoon_Draw(thisx, globalCtx);
 }
 
-void EnFall_FireBall_Draw(Actor* thisx, GlobalContext* globalCtx) {
+void EnFall_Fireball_Draw(Actor* thisx, GlobalContext* globalCtx) {
     s32 pad;
     EnFall* this = THIS;
     u32 gameplayFrames = globalCtx->gameplayFrames;
 
     OPEN_DISPS(globalCtx->state.gfxCtx);
+
     gSPMatrix(POLY_XLU_DISP++, Matrix_NewMtx(globalCtx->state.gfxCtx), G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
     func_8012C2DC(globalCtx->state.gfxCtx);
-    this->fireBallYTexScroll1 += (s32)(4.0f + (this->fireBallIntensity * 12.0f));
-    this->fireBallYTexScroll2 += (s32)(2.0f + (this->fireBallIntensity * 6.0f));
+
+    this->fireballYTexScroll1 += (s32)(4.0f + (this->fireballIntensity * 12.0f));
+    this->fireballYTexScroll2 += (s32)(2.0f + (this->fireballIntensity * 6.0f));
+
     gDPSetPrimColor(POLY_XLU_DISP++, 0, 0x80,
-                    (s32)(((1.0f - this->fireBallIntensity) * 160.0f) + (255.0f * this->fireBallIntensity)),
-                    (s32)((70.0f * (1.0f - this->fireBallIntensity)) + (255.0f * this->fireBallIntensity)),
-                    (s32)(70.0f * (1.0f - this->fireBallIntensity)), 255);
+                    (s32)(((1.0f - this->fireballIntensity) * 160.0f) + (255.0f * this->fireballIntensity)),
+                    (s32)((70.0f * (1.0f - this->fireballIntensity)) + (255.0f * this->fireballIntensity)),
+                    (s32)(70.0f * (1.0f - this->fireballIntensity)), 255);
     gDPSetEnvColor(
-        POLY_XLU_DISP++, (s32)(((1.0f - this->fireBallIntensity) * 50.0f) + (200.0f * this->fireBallIntensity)),
-        (s32)(20.0f * (1.0f - this->fireBallIntensity)), (s32)(20.0f * (1.0f - this->fireBallIntensity)), 255);
+        POLY_XLU_DISP++, (s32)(((1.0f - this->fireballIntensity) * 50.0f) + (200.0f * this->fireballIntensity)),
+        (s32)(20.0f * (1.0f - this->fireballIntensity)), (s32)(20.0f * (1.0f - this->fireballIntensity)), 255);
 
-    // Glowing sphere of fire
+    // Handles the texture scrolling for the glowing sphere of fire
```

Comment:
```suggestion
    // For the glowing sphere of fire
```
I think this would probably be sufficient.